src/Components/Pages/Contactus.js

```diff
@@ -56,91 +55,100 @@ const Contactus = () => {
 	const { error, loading, clearErrors, showLoading } = authContext;
 
 	const [details, setDetails] = useState({
-		Firstname: '',
-		Lastname: '',
-		Email: '',
-		Mobile: '',
-		Reason: ''
+		Firstname: "",
+		Lastname: "",
+		Email: "",
+		Mobile: "",
+		Reason: "",
 	});
 
 	const { FirstName, LastName, Email, Mobile, Reason } = details;
 
-	const handleChange = e => {
+	const handleChange = (e) => {
 		setDetails({
 			...details,
-			[e.target.name]: e.target.value
+			[e.target.name]: e.target.value,
 		});
 	};
 
-	const submitContact = async e => {
-		console.log('called submit contact');
+	const submitContact = async (e) => {
+		console.log("called submit contact");
 		e.preventDefault();
 		showLoading({ data: true });
 		let contactData = {
-			name: FirstName + ' ' + LastName,
+			name: FirstName + " " + LastName,
 			email: Email,
 			contact: Mobile,
-			query: Reason
+			query: Reason,
 		};
 
 		try {
-			let res = await axios.post(backendUrl + '/api/contact', contactData);
-			if (res.data.msg == 'Query submitted successfully') {
-				setAlert(res.data.msg, 'success');
+			let res = await axios.post(
+				backendUrl + "/api/contact",
+				contactData
+			);
+			if (res.data.msg == "Query submitted successfully") {
+				setAlert(res.data.msg, "success");
 				setDetails({
-					FirstName: '',
-					LastName: '',
-					Email: '',
-					Mobile: '',
-					Reason: ''
+					FirstName: "",
+					LastName: "",
+					Email: "",
+					Mobile: "",
+					Reason: "",
 				});
 			} else {
-				setAlert(res.data.msg, 'danger');
+				setAlert(res.data.msg, "danger");
 			}
 			clearErrors();
 		} catch (err) {
-			console.log('Error:', err);
+			console.log("Error:", err);
 			setAlert(
-				'Error occured while submitting your query.Please try again or write to us at technojam@galogtiasuniversity.edu.in',
-				'danger'
+				"Error occured while submitting your query.Please try again or write to us at technojam@galogtiasuniversity.edu.in",
+				"danger"
 			);
 			clearErrors();
 			showLoading({ data: false });
 		}
 	};
 
 	return (
-		<Container maxWidth='lg'>
+		<Container maxWidth="lg">
 			<br />
 			<div style={sectionStyle}>
 				<Grid
 					container
-					direction='row'
-					justify='flex-start'
-					alignItems='flex-start'
+					direction="row"
+					justify="flex-start"
+					alignItems="flex-start"
 					spacing={6}
 				>
-					<Grid direction='row' xs={12} sm={12} md={6} style={{padding:"20px"}}>
+					<Grid
+						direction="row"
+						xs={12}
+						sm={12}
+						md={6}
+						style={{ padding: "20px" }}
+					>
 						<Card
-							className='card--shadow'
-							style={{ borderRadius: '10px', }}
+							className="card--shadow"
+							style={{ borderRadius: "10px" }}
 						>
-							<div style={{ backgroundColor: '#24292E' }}>
```

Comment:
we don't want double-quotes. Move this CSS in separate file

Reply:
> we don't want double-quotes. Move this CSS in separate file

Calm down
You know you used material-ui framework
Not all styles can be moved to a different file